packages/preview/oh-my-euxfel/1.0.0/README.md

```diff
@@ -0,0 +1,31 @@
+# An XFEL Polylux theme
+
+This is a [Polylux](https://polylux.dev/book/) theme for [European
```

Comment:
Polylux is not very actively maintained and the latest version is using deprecated functions. I would rather not merge a package that reports deprecation warnings as they will become hard errors with the next Typst release. I contacted the Polylux maintainer to ask them to make a new release to fix that, but it may take some time before they actually do it. If you would like to have your package released rapidly, I would suggest switching to another presentation package, like `touying` which should have a similar API and that is more actively maintained.

Reply:
Ah I see, I wasn't aware of that... I'll have a go at switching it.

Reply:
Actually, I already got an answer from them and they said they would make a release to fix this issue soon. So if you prefer to wait a few days, it should not be *that* long.

Reply:
Polylux 0.4.0 was just released, updating it should fix the issues.

Reply:
Thanks for the reminder, I forgot about this :sweat_smile: I have heard good things about touying though so I'll have another look at that before proceeding, will try to finish this this week.